src/os/data/layernode.js

```diff
@@ -42,6 +42,7 @@ os.data.LayerNode = function() {
   this.layer_ = null;
 
   os.ui.queryManager.listen(goog.events.EventType.PROPERTYCHANGE, this.onNodeChanged_, false, this);
+  os.dispatcher.listen(os.im.action.ImportActionEventType.REFRESH, this.onNodeChanged_, false, this);
```

Comment:
This can simply listen on the `ImportActionManager`. Most of the manager classes in OS extend `goog.events.EventTarget`. That base class is where all these classes inherit the `listen/unlisten/dispatchEvent` methods, and we prefer to keep listeners off of `os.dispatcher` as much as possible.

Also, it needs to unlisten in the `disposeInternal` method below, otherwise the `LayerNode` instances will be retained in memory because the listener reference still exists.